eva/executor/orderby_executor.py

```diff
@@ -65,14 +73,30 @@ def exec(self) -> Iterator[Batch]:
             aggregated_batch_list.append(batch)
         aggregated_batch = Batch.concat(aggregated_batch_list, copy=False)
 
+        # Column can be a functional expression, so if it
+        # is not in columns, it needs to be re-evaluated.
+        merge_batch_list = [aggregated_batch]
+        for col in self._columns:
```

Comment:
Can we use the `extract_column_names ` function? Looks like duplicate code

Reply:
Simplify this.